Source/Particles/AddPlasmaUtilities.H

```diff
@@ -0,0 +1,209 @@
+/* Copyright 2024 Andrew Myers
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
```

Comment:
Have to unify it another time, but this is slightly better for new files for now:
```suggestion
/* Copyright 2024 The WarpX Community
 *
 * This file is part of WarpX.
 *
 * License: BSD-3-Clause-LBNL
 * Authors: Andrew Myers
```